wolfssh/internal.h

```diff
@@ -467,6 +479,7 @@ typedef struct HandshakeInfo {
 #endif
 
     byte useEcc;
+    byte useSaber;
```

Comment:
Please wrap with `#ifndef WOLFSSH_NO_SABER_LEVEL1_SHA256`. Also these should be bit-fields. `byte useSaber:1;` and `byte useEcc:1;`. You'll also need all the code with `!ssh->handshake->useSaber)` wrapped in the macro. I don't want code size to grow if saber is not used.

Reply:
Do use a bit-field with useSaber, but don't change useEcc yet. That's off topic for this PR. Doing it right when adding it is good. I don't want to change things piecemeal. I'd rather we convert the flags all at once. (The single bit flags aren't the worst memory wasters in wolfSSH, anyway.)

Reply:
OH...thats an interesting point.  If this gets expanded to different levels (ie 1, 3, 5) then I can switch this back to BYTE.